yaksh/templates/yaksh/course_modules.html

```diff
@@ -7,14 +7,14 @@
     <div class="card">
         <div class="card-header">
             {{ course.name }}
-            <a href="{% url "yaksh:course_forum" course.id %}" class="btn btn-info pull-right">Discussion Forum</a>
+            <a href="{% url 'yaksh:course_forum' course.id %}" class="btn btn-info pull-right">Discussion Forum</a>
         </div>
         <div class="card-body">
-            {% if course.view_grade %}
+            
                 <center>
-                    <b>Grade:</b> {% if grade %} {{ grade }} {% else %} Will be available once the course is complete {% endif %}
+                    <b>Grade:</b> {% if grade %} {{grade}} {% else %} Will be available once the course is complete {% endif %}
                 </center>
-            {% endif %}
```

Comment:
I guess you don't have to remove this if condition, because the course creator has the control to enable and disable to view the grades for the course.